examples/basic.js

```diff
@@ -19,14 +19,10 @@ const accessToken = (() => {
   return process.argv[2];
 })();
 
-const actions = {
-  send(request, response) {
-    const {sessionId, context, entities} = request;
-    const {text, quickreplies} = response;
-    console.log('user said...', request.text);
-    console.log('sending...', JSON.stringify(response));
-  },
-};
+const handleMessage = (rsp) => {
```

Comment:
just remove since we have a default handleMessage behavior in `interactive` (or move this there)